render/render.go

```diff
@@ -3,16 +3,24 @@ package render
 import (
 	"strings"
 	"sync"
+	"sync/atomic"
 	"syscall"
 	"time"
 
 	"github.com/aaronjanse/3mux/ecma48"
 )
 
+// This is arbitrarily chosen. Once we reach two thirds of this, we switch to frame-based rendering
+const queueBufferSize = 4000
```

Comment:
Should we allow custom buffers? (like an option)

Reply:
I think so. This will require some UI considerations because we don't want to expose users to configuration options that can't be judged without jumping into Alacritty/Kitty/xterm source code